docs/.gitignore

```diff
@@ -12,6 +12,7 @@ config.local.js
 basement_dist
 .idea
 report-check-links.xlsx
+react-translation-log.html
```

Comment:
According to the project requirements, this file should not be ignored:

> This file will be committed to the repo to allow detecting regressions in the transformation script.

This is especially important since `snippet-transform` does not have a test suite.

Reply:
I generated the initial translation log and committed in here: 5b558ceb1a9475452e55019cea4f1358938dc73b